cmp/viewmanager/DataAccess.ts

```diff
@@ -48,14 +48,14 @@ export class DataAccess<T> {
     }
 
     /** Fetch the latest version of a view. */
-    async fetchViewAsync(info: ViewInfo): Promise<View<T>> {
+    async fetchViewAsync(token: String): Promise<View<T>> {
         const {model} = this;
-        if (!info) return View.createDefault(model);
+        if (!token) return View.createDefault(model);
         try {
-            const raw = await XH.fetchJson({url: 'xhView/get', params: {token: info.token}});
+            const raw = await XH.fetchJson({url: 'xhView/get', params: {token}});
```

Comment:
I wonder if the params here should also include `type` from ViewManagerModel, so that a token for a blob not related to this view manager cannot be fetched.  (this risk precedes this change)

Reply:
could also just throw in View.fromBlob, if it does not match -- easier change.